### PR TITLE
What's New 67 page

### LINKED ISSUE #6799: WNP 67- Coding Design - Release date - 5/14
[Master Project Document](https://docs.google.com/document/d/1_Jb3Z7RpVM-o5YKqOKDgW4v5CMdzSGIPTDa5GtYg-8A/edit)

| Estimates |  |
| ------------- | ------------- |
| TL | |
| PM | 2 days |
| ENG | |
| ANA | TBD |
| DES | TBD |
| COPY | TBD |

Description: launch the whatsnew page one week before the Firefox 67 launch to promote FxA accounts (and possible other things).

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Live and ready for QA (Soft Vision) on or before - 5/7

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] New copy string delivery in time to maximize l10n reach

#### Tasks
- [x] File bug requesting WNP w. Releng (Eric R)
- [x] Copy (Meghan)
- [x] Design (Carolyn)
- [ ] Code (Stephanie)
- [ ] Provide data reporting on CTA metrics (Saptarshi)

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

Strings, code, and assets for What's New 67.

## Issue / Bugzilla link

Fix #6799

## Testing

When signed in to FxA and in a supported locale the newsletter sign up form displays.

### CODE REVIEW

File: media/css/firefox/whatsnew/whatsnew-67.scss [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+@import '../../../protocol/css/components/feature-card';
+@import '../../../protocol/css/components/hero';
+@import '../../../protocol/css/components/modal';
+@import '../../../protocol/css/components/newsletter-form';
+
+@import '../../protocol/components/state-fxa';
+
+
+main {
+    color: $color-off-black;
```
Comment (alexgibson) [inline media/css/firefox/whatsnew/whatsnew-67.scss]: Not a blocker, but could this be something we land in Protocol first? 

I know this is what the mockup shows, but I think we need to start being more concious of creating inconsistency and tech-debt across pages. This feels somewhat non-critical, and something that could be introduced with little friction site wide, as opposed to on a page by page basis that we then clean up later?

File: media/css/firefox/whatsnew/whatsnew-67.scss [hunk 2]
```diff
@@ -0,0 +1,500 @@
...
+
+//* -------------------------------------------------------------------------- */
+// Up-to-date page header
+
+.c-page-header-up-to-date {
+    @include text-display-xxs;
+    display: none;
+
+    span {
+        background-color: #6EE7C4;
```
Comment (alexgibson) [inline media/css/firefox/whatsnew/whatsnew-67.scss]: Should this use a token in the Protocol color pallet? I think we've seen this type of notification in the designs for /thanks, so it seems like something that will appear in multiple places.

File: media/css/firefox/whatsnew/whatsnew-67.scss [hunk 3]
```diff
@@ -0,0 +1,500 @@
...
+    margin: 0 auto $layout-sm auto;
+    text-align: center;
+}
+
+.mzp-c-card-feature.mzp-l-card-feature-right-66.wn67-intro {
+    margin-bottom: $layout-md;
+    max-width: $content-lg;
+
+    .mzp-c-card-feature-title {
+        @include visually-hidden;
```
Comment (alexgibson) [inline media/css/firefox/whatsnew/whatsnew-67.scss]: Curious why this title is hidden visually but not from screen readers. Would it be worth adding a comment to explain?

File: media/css/firefox/whatsnew/whatsnew-67.scss [hunk 4]
```diff
@@ -0,0 +1,500 @@
...
+
+.mzp-c-card-feature {
+    margin-bottom: $layout-md;
+
+    .mzp-c-card-feature-media img {
+        max-width: 300px;
+        margin: 0 auto;
+
+        [dir='rtl'] & {
+            transform: scaleX(-1);
```
Comment (alexgibson) [inline media/css/firefox/whatsnew/whatsnew-67.scss]: 🔥

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.html [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+      include_cta=False
+    ) %}
+    {% endcall %}
+    -->
+
+    {% call feature_card(
+      title=_('New to your browser'),
+      image_url='firefox/whatsnew_67/update.svg',
+      include_highres_image=False,
+      aspect_ratio='mzp-l-card-feature-right-66',
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.html]: If this design requires non-standard layout classes, suggest not using the `mpl` namespace?

**Comment by alexgibson** [comment]
The update notification is a little close to the Firefox wordmark at medium and small viewport sizes:

image

image

**Comment by alexgibson** [comment]
I'm assuming this is a bug, but if I close the accounts CTA I get presented with the newsletter CTA? 🤪

newsletter

**Comment by alexgibson** [comment]
Re above: my mistake, I see what's happening. It's not the newsletter but the accounts form still. I happened to be at the end of the page when I clicked close, so got mega-confused when only the close button disappeared 😂

I wonder how many people this might happen to, seeing as the newsletter is not stuck to the bottom of the viewport. Could it be worth really hiding it if someone dismisses it?

File: media/css/firefox/whatsnew/whatsnew-67.scss (hunk 2 above)
Comment (stephaniehobson) [reply media/css/firefox/whatsnew/whatsnew-67.scss]: It's not a colour that 's in the pallet right now 😠 

I do think it's something we should add.

**Comment by stephaniehobson** [comment]
I think the stakeholders will object to the form being removed when dismissed. I'll look into removing the "X" when the form is near the bottom of the page.

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.html [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+    <div class="c-page-header-inner">
+      {{ high_res_img('firefox/whatsnew_67/logo.png', {'alt': 'Firefox', 'width': '147', 'height': '55', 'class': 'c-page-header-logo-fx'}) }}
+      <h1 class="c-page-header-up-to-date"><span>{{ _('Congrats! You’re using the latest version of Firefox.') }}</span></h1>
+    </div>
+  </header>
+
+  <div class="mzp-l-content">
+    {# L10n: "More power to you" is an American English idiom wishing someone success in their endeavors, and here is also a play on words saying Firefox gives users more power and control over their web experience. A more literal alternative would be "You have more power with every update" #}
+    <h2 class="c-page-title">{{ _('More power to you with every update') }}</h2>
+    <!--
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.html]: ✅

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.html [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+{% extends "firefox/base-protocol.html" %}
+
+{% block page_title %}{{ _('Get a Firefox Account – Keep your data private, safe and synced') }}{% endblock %}
+{% block page_desc %}{{ _('Securely sync your passwords, bookmarks and tabs across all your devices. Get a Firefox Account now – One login – Power and privacy everywhere.') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_67') }}
+{% endblock %}
+
+{% set fxa_utm_params = {
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.html]: It was not but now it is.

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.html [hunk 8]
```diff
@@ -0,0 +1,190 @@
+{# License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.html]: ✅

File: media/css/firefox/whatsnew/whatsnew-67.scss (hunk 1 above)
Comment (stephaniehobson) [reply media/css/firefox/whatsnew/whatsnew-67.scss]: I agree with you. And the cat is already out of the bag on this one, there's several pages with the new colour and it was approved by stakeholders.

File: media/css/firefox/whatsnew/whatsnew-67.scss (hunk 3 above)
Comment (stephaniehobson) [reply media/css/firefox/whatsnew/whatsnew-67.scss]: I thought I needed to preserve the heading order but on closer inspection it's not necessary.

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.html (hunk 5 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.html]: ✅

**Comment by stephaniehobson** [comment]
I don't think I have time to fix the bug on the newsletter signup form 😞 I can't think of a quick way to hide the "X" that doesn't involve adding a bunch of JS to this page. (IntersectionObserver is not so good with handling scrolling back and fourth in my experience.) And I don't have time to go back to the stakeholders, who have been very adamant that this form is the only thing of value to them, to try to negotiate removing it.

My recommendation would be to merge so QA can begin their work and open a separate bug to try to address the newsletter form later. Thoughts @alexgibson?

**Comment by ejregithub** [comment]
+1 to getting this out as is, then discussing an iteration to address the newsletter form concern.

File: media/css/firefox/whatsnew/whatsnew-67.scss (hunk 1 above)
Comment (alexgibson) [reply media/css/firefox/whatsnew/whatsnew-67.scss]: I’m guessing time is of the essence here and we’re gonna ship with this, but fwiw I think we can change things like this if we decide it’s the right thing to do, even after stakeholders have approved a mock-up. Let’s chalk this small palate cut up and lay that debt later 😉

**Comment by alexgibson** [comment]
I’ll file an issue for the newsletter form. Agree it’s not a blocker but let’s see if we can fix the UX, as i’m probably not the only one who will get confused by it. 👍

**Comment by alexgibson** [comment]
Filed an issue for the sticky form: https://github.com/mozilla/bedrock/issues/7134

Also commented on https://github.com/mozilla/protocol/issues/372 and asked for some clarify from @dzingeek on what the new type color rules should be.

**Comment by dzingeek** [comment]
@alexgibson are there still questions around type colors? that card you mentioned above was closed. If there are please reach out to @mrkwvr as he has been updating the brand guidelines. He can point you to the correct info.

**Comment by alexgibson** [comment]
@dzingeek It looks like @stephaniehobson closed the issue, so I think we can assume she has the info that was needed.

Where can we see the updated brand guidelines?

**Comment by stephaniehobson** [comment]
Well I just guessed, we should probably check I guessed right.

**Pull request merged**